apps/admin/frontend/src/app_root.tsx

```diff
@@ -53,27 +48,28 @@ export function AppRoot({
   const { cardReader, printer: printerInfo } = useDevices({ hardware, logger });
 
   const [isTabulationRunning, setIsTabulationRunning] = useState(false);
-  const [manualTallyVotingMethod, setManualTallyVotingMethod] = useState(
-    VotingMethod.Precinct
-  );
```

Comment:
We agreed to just remove the toggling voting method feature temporarily as it's not relevant now: https://votingworks.slack.com/archives/CEL6D3GAD/p1683914471084739.